modules/mtbseq/tbamend/tbamend.nf

```diff
@@ -0,0 +1,49 @@
+nextflow.enable.dsl = 2
+// NOTE: To properly setup the gatk inside the docker image
+// - Download the gatk-3.8.0 tar file from here https://console.cloud.google.com/storage/browser/gatk-software/package-archive/gatk;tab=objects?prefix=&forceOnObjectsSortingFiltering=false
+// - tar -xvf GATK_TAR_FILE
+// - gatk-register gatk_folder/gatk_jar
+
+
+params.resultsDir = "${params.outdir}/tbamend"
+params.saveMode = 'copy'
+params.shouldPublish = true
+
+process TBAMEND {
+    tag "${project_name}"
+    publishDir params.resultsDir, mode: params.saveMode, enabled: params.shouldPublish
+
+    input:
+    tuple path(samples_file), val(project_name), path("${params.mtbseq_project_name}_*_samples.tab")
+    path(gatk_jar)
+    env USER
+
+    output:
+    path("Amend/*")
+    tuple val(project_name), path  ("Amend/*_joint_*_samples_amended_*_phylo_*.tab"), emit: next_step
+
+    script:
+
+    """
+
+    gatk-register ${gatk_jar}
+
+    mkdir Amend
+    MTBseq --step TBamend --samples ${samples_file} --project ${project_name}
+
+    """
+    stub:
+    """
+    mkdir Amend
+    touch Amend/${params.mtbseq_project_name}_joint_[mincovf]_[mincovr]_[minfreq]_[minphred20]_samples_amended.tab
```

Comment:
@Mxrcon , please understand that the `[mincovf]` etc are supposed to have actual values and this is were the lack of actual execution bites us with stubbing. 

I'll share a folder with previous runs with you for you to understand the exact names of the files which are generated. 

Please update all the stubs once you understand how the actual data looks like.

Reply:
I've shared a folder with you from a onedrive account.

Reply:
@abhi18av hey, I haven't received any folder, I think that i solved all the other problems, I'll wait for the files to fix this and then request a new review